src/cordova/apple/xcode/ios/Outline/VpnExtension-Info.plist

```diff
@@ -17,9 +17,9 @@
 	<key>CFBundlePackageType</key>
 	<string>XPC!</string>
 	<key>CFBundleShortVersionString</key>
-	<string>1.7.0</string>
```

Comment:
We never released 1.7.1, so we could keep that number.

Reply:
It doesn't compile unless the versions match